### ISSUE TITLE
Feature request: launch seperate apps from xochitl

### ISSUE THREAD

**Issue by iHnR** [ddvk/remarkable-hacks#397]
Seperate launchers are a bit messy. The xochitl sidebar seems to have all the necessary real estate add some app buttons. I understand it's probably far beyond the scope of this project but it would be great to have.

**Comment by ddvk** [comment]
yes, far beyond, also the whole screen / input management has to be done somewhere

**Comment by martin-braun** [comment]
I like the idea from a customers point of view, but I agree that this is very far of what is currently possible.
